app/views/users/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with model: @user, local: true do |f| %>
+  <%= f.label :email, 'Email:' %>
+  <%= f.text_field :email %>
+  <%= f.label :password, 'Password:' %>
+  <%= f.password_field :password %>
+  <%= f.label :password_confirm, 'Confirm Password:' %>
+  <%= f.password_field :password_confirm %>
```

Comment:
After class I was wondering how would this work since we didn't touch base. Will have to go over it for me team!